source/configuration/collection-config.txt

```diff
@@ -0,0 +1,104 @@
+.. _mongoid-collection-config:
+
+========================
+Collection Configuration
+========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, collections, time series, capped collection
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to specify configuration options for a
+collection in your {+odm+} application.
+
+Configure Collection Options
+----------------------------
+
+You can specify configuration options for a collection by using the
+``:collection_options`` argument with the ``store_in``
+macro. The ``:collection_options`` argument accepts any collection option that
+your driver and server version supports. 
+
+.. note::
+
+   To apply the collection options, your collection must be explicitly created
+   prior to use. To learn how to create a collection using a Rake task, see the
+   :ref:`mongoid-create-collection-rake` section of this guide.
```

Comment:
S: consider linking to docs in server or atlas about creating collections.

Reply:
Ended up rewording this section to better match what the original docs were saying. It says that it "should be done" by the Rake task so it seems to me like they want to point users that way rather than other ways of creating collection